packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -29,6 +31,20 @@ type CreateOptions = {
   input: TCreateInputSchema;
 };
 
+type LoggedInUserType = {
```

Comment:
nit: I think we should be fine with Type suffix in it as it can be used in type context only, so it is clear that it is a type.